snmp/tests/README.md

```diff
@@ -0,0 +1,17 @@
+# SNMP Testing
+
+## Known issues
+
+### macOS/Windows: "No SNMP response received before timeout for instance localhost"
+
+This is most likely due to a bug with cross-container UDP communication in Docker for macOS and Windows.
+
+A workaround is to pass set your IPv4 host address as the `DD_SNMP_HOST` environment variable, e.g.:
```

Comment:
On which interface exactly? Would `host.docker.internal` work as a better workaround?
https://docs.docker.com/docker-for-mac/networking/